src/Cool/DidYouKnow/testDidYouKnowModal.cpp

```diff
@@ -0,0 +1,32 @@
+#include "Cool/DidYouKnow/testDidYouKnowModal.hpp"
+#include <fmt/chrono.h>
+#include <imgui.h>
+
+void did_you_know(Cool::DidYouKnowModal& _did_you_know)
+{
+    _did_you_know.open_ifn();
+
+    _did_you_know.imgui_windows();
+}
+
+namespace Cool {
+
+void test_did_you_know(Cool::DidYouKnowModal& _did_you_know)
+{
+    if (ImGui::Button("Test DidYouKnow"))
+    {
+        _did_you_know.open();
+    }
+
+    _did_you_know.imgui_windows();
+
+    // imgui text with difference between current timestamp of did you know and current timestamp of now
+    ImGui::TextUnformatted(fmt::format("Difference: {}", (std::chrono::steady_clock::now() - _did_you_know._timestamp_last_opening)).c_str());
```

Comment:
A more useful thing to display in order to debug would be the time remaining until the next "Did You Know". Also, you can format the time in a nicer way with https://hackingcpp.com/cpp/libs/fmt.html#chrono